src/test/DoS_tests.cpp

```diff
@@ -66,11 +66,14 @@ BOOST_AUTO_TEST_CASE(outbound_slow_chain_eviction)
     dummyNode1.fSuccessfullyConnected = true;
```

Comment:
In tests where no multi-threading happens (most of them) it would work just as well to just take the necessary locks once in the beginning of each test case. That would make the diff smaller likely.

Reply:
@laanwj Do you know which tests in this PR that do not have any multi-threading? What is the best way to identify the tests for which multi-threading happens?

(Personally I like explicit locking better since that communicates where/why the lock is needed which is helpful for locking newcomers. I'll happily adjust to consensus however :-))

Reply:
Yeah I'd say just keep it like this...